### ISSUE TITLE
Add tests for dnsrecords weights

### ISSUE THREAD

**Issue by ficap** [Kuadrant/testsuite#117]
See https://github.com/Kuadrant/kcp-glbc/commit/7c23cbe7c9e03703e8e11be8d0df84aa39ef9a06

**Comment by pehala** [comment]
Does this affect a behaviour customers would see?

**Comment by ficap** [comment]
Yes it does. It may affect performance seen by customer (load balancing). It may not be easy to implement from black box point of view though.

**Comment by pehala** [comment]
> It may not be easy to implement from black box point of view though.

That's exactly the problem. Currently, we expect to test a completely black box system, and customers won't have access to AWS route53 settings.

**Comment by ficap** [comment]
But they do have access to DNSRecord CR where they can set this to their needs manually too.

**Comment by jsmolar** [comment]
@ficap please update this issue

**Comment by trepel** [comment]
discussed on Apr 29:
@trepel suggested that testing this on DNS Operator level checking that weight makes it from DNSRecord CR to actual DNS record in DNS provider should suffice and there's no need to have an e2e test for this in testsuite.
Alternative is some statistical approach sending many requests, can be improved by different weight values and TTL values. It might be tricky to see what cluster was used to serve the request. @azgabur @averevki might chime in here.

No conclusion yet.
